### PR TITLE
Feature query update

### PR DESCRIPTION
**Pull request by uisendy**
In this milestone,  transactions to update and delete records were performed. aggregate functions and GROUP BY were also used to answer analytical questions.

The following was performed carefully in this activity. 

1. Add a new Column to the Animal Table and called it "species"
2. Insertion of more data into the animal table. The total Data after insertion is 11. 
3. Use Transaction to perform a specific task with the table concerning the update, query, deletion, rollover, savepoint and commit, 
4. Using Aggregate functions to answer the specific questions concerning the take. See the queries.sql for the questions in the comment and the specific queries. 

**Screenshots of play and queries**

aggregate-1
aggregate-2
aggregate-3
aggregate-4
aggregate-5
aggregate-7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -5,3 +5,13 @@ VALUES  ('Agumon', '2020-02-03', 0, true, 10.23),
         ('Gabumon', '2018-11-15', 2, true, 8),
         ('Pikachu', '2021-01-07', 1, false, 15.04),
         ('Devimon', '20017-05-12', 5, true, 11)
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES  ('Charmander', '2020-02-08', 0, false, 11),
+        ('Plantmon', '2021-11-15', 2, true, 5.7),
+        ('Squirtle', '1993-04-02', 3, false, 12.13),
+        ('Angemon', '2005-06-12', 1, true, 45)
```
Comment (RolandM99) [inline data.sql]: - [ ] The weight of these animals should be negative. kindly, change their values into negative values.

File: queries.sql [hunk 2]
```diff
@@ -24,4 +24,85 @@ SELECT * FROM animals WHERE neutered = true
 SELECT * FROM animals WHERE name != 'Gabumon'
 
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Inside a Transaction
```
Comment (RolandM99) [inline queries.sql:29]: - [ ] Kindly, create the queries to update the `animals` table by setting the species column to unspecified, then roll back the change. Take a look on the code snippet below:
```sql
BEGIN; -- start transaction
   UPDATE animals
   SET species = 'unspecified';
   SELECT species from animals; 
   ROLLBACK;
   SELECT species from animals;
```

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @uisendy  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @uisendy 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — COMMENTED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @uisendy , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice query🔥 
✅ Linters are passing 👏 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**